README.md

```diff
@@ -53,17 +53,3 @@ This is our recommended 2.5 build if you don't need `qt5` or `xvfb`
 - Node: Latest 12.x
 - Postgres Client: Latest 9.6.x
 - QT: Latest available in stretch
-
-## Ruby 2.4
-
-*Ruby 2.4 is end of life, DO NOT USE*
-
-### 2.4-alpine
-
-- Node: uses the latest node available for alpine 3.7
-- Postgres Client: uses the latest postgres-client available for alpine 3.7
-
-### 2.4-stretch-slim
-
-- Node: Latest 8.x
-- Postgres Client: Latest 9.6.x
```

Comment:
I removed this from the README because it appeared to be vestigial as these images don't exist in the repo anymore.